OpenSim/Simulation/Manager/Manager.h

```diff
@@ -208,9 +218,11 @@ class OSIMSIMULATION_API Manager
     //--------------------------------------------------------------------------
     // EXECUTION
     //--------------------------------------------------------------------------
+    bool integrate(SimTK::State& s, double finalTime);
```

Comment:
Add a doxygen comment for this function, since we expect this to be the one people actually use.